app/assets/javascripts/controllers/cloud_network/cloud_network_form_controller.js

```diff
@@ -56,4 +56,18 @@ ManageIQ.angular.app.controller('cloudNetworkFormController', ['$scope', 'cloudN
     $scope.angularForm.$setPristine(true);
     miqService.miqFlash("warn", "All changes have been reset");
   };
+
+  $scope.filterNetworkManagerChanged = function(id) {
+    miqService.sparkleOn();
+
+    API.get("/api/providers/" + id + "/cloud_tenants?expand=resources&attributes=id,name")
+      .then(getCloudTenantsByEms)
+      .catch(miqService.handleFailure);
+
+    miqService.sparkleOff();
```

Comment:
`sparkleOff` here doesn't make sense, calling `sparkleOn` and `sparkleOff` in the same tick pretty much means it will never start to spin.

I guess you want to `sparkleOff` in `getCloudTenantsByEms`.